CCS/Guadaloop_lib/guadaloop/lib/sensors/queueData.h

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
thoroughly document all of this file with doxygen notation. Also add comments to all the *.c file you made changes to.